astro/src/content/docs/apis/identity-providers/_external-jwt-request-body.mdx

```diff
@@ -84,7 +84,13 @@ import IdentityProviderLinkingStrategyRequestParameter from 'src/content/docs/ap
   </APIField>
 
   <IdentityProviderTenantConfiguration />
+  <APIField name="identityProvider.tenantId" type="UUID" optional since="1.62.0">
+    The unique Id of the Tenant.
 
+    Providing a value creates an identity provider scoped to the specified tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified tenant while global identity providers can be used with any tenant.
```

Comment:
```suggestion
    Providing a value creates an identity provider scoped to the specified tenant, otherwise a global identity provider is created. Tenant-scoped identity providers can only be used to authenticate in the context of the specified tenant. Global identity providers can be used with any tenant.
```
[optional]: same as above, break up long sentence for readability's sake